ui/packages/consul-ui/app/components/route/title/index.hbs

```diff
@@ -0,0 +1,15 @@
+{{page-title @title separator=@separator}}
+{{#if (not-eq @render false)}}
+{{@title}}
+{{/if}}
+<Portal @target="route-announcer">
+<div
+  class="route-title"
+  ...attributes
+  aria-live="assertive"
+  aria-atomic="true"
+>
+  {{! Using a handlebars concat here avoid whitespace issues}}
+  {{concat 'Navigated to ' @title}}
```

Comment:
The whitespace that is added by Handlebars when you do

```hbs
Navigated to {{@title}}
{{! yields (note the carriage return) }}
Navigated to
Title of page
```

...is noticeable when this is read out. Concat'ing this here works around that.